draft-packages/modal/docs/InputEditModal.stories.tsx

```diff
@@ -98,7 +108,8 @@ InputEditModalExample.storyName = "Input Edit Modal"
 InputEditModalExample.args = {
   mood: "positive",
   title: "Input-edit modal title",
-  submitLabel: "Submit label",
+  submitLabel: "Primary label",
+  secondaryLabel: "Secondary label",
```

Comment:
Given this is more of an edge case and we only have one story for this modal, I'm thinking this story shouldn't have the secondary action by default.